### PR TITLE
scio 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by nevillelyh**
- [ ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [ ] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

File: Formula/scio.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+              java_opts+=("$o")
+          else
+              scio_opts+=("$o")
+          fi
+      done
+
+      exec java ${java_opts[*]} -jar #{libexec}/"scio-repl-#{version}.jar" ${scio_opts[*]}
+    EOS
+  end
+end
```
Comment (MikeMcQuaid) [inline Formula/scio.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Formula/scio.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      #!/bin/bash
+      java_opts=()
+      scio_opts=()
+      for o in "$@"; do
+          if [[ "$o" =~ ^-D.* ]]; then
+              java_opts+=("$o")
+          else
+              scio_opts+=("$o")
+          fi
+      done
```
Comment (MikeMcQuaid) [inline Formula/scio.rb]: Please submit this wrapper script logic to the upstream project, it's too complicated to live in a Homebrew formula.

File: Formula/scio.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+      exec java ${java_opts[*]} -jar #{libexec}/"scio-repl-#{version}.jar" ${scio_opts[*]}
+    EOS
```
Comment (MikeMcQuaid) [inline Formula/scio.rb]: Use `bin.write_jar_script` instead.

File: Formula/scio.rb (hunk 2 above)
Comment (MikeMcQuaid) [inline Formula/scio.rb]: I see you have upstream commit access (or at least previously merged commits) so extra 👍 on this.

**Comment by MikeMcQuaid** [comment]
> No description provided.

Please fill out the PR template, it's there for a reason, thanks.

**Comment by nevillelyh** [comment]
Thanks. We decided to keep it in our own tap.

**Pull request closed without merging**